### ISSUE TITLE
Dead link "predict" method

### ISSUE THREAD

**Issue by filipre** [mlr-org/mlr-tutorial#33]
I found a dead link inside the 2.4 as well as 2.7 tutorial https://mlr-org.github.io/mlr-tutorial/release/html/create_learner/index.html for the `predict` method (see code).

Code: 
- https://github.com/mlr-org/mlr-tutorial/blob/gh-pages/2.4/html/create_learner/index.html
- https://github.com/mlr-org/mlr-tutorial/blob/gh-pages/2.7/html/create_learner/index.html

```
<p>The definition for LDA looks like this. It is pretty much just a straight
pass-through of the arguments to the <a href="http://www.inside-r.org/r-doc/base/predict">predict</a> function and some extraction of
prediction data depending on the type of prediction requested.</p>
```

**Comment by filipre** [comment]
https://mlr-org.github.io/mlr-tutorial/release/html/multilabel/index.html#predict 

```
Prediction can be done as usual in mlr with predict and by passing a trained model and either the task to the task argument or some new data to the newdata argument. As always you can specify a subset of the data which should be predicted.
```

**Comment by berndbischl** [comment]
Thanks. 
@schiffner Can we quickly fix this and close?

**Comment by schiffner** [comment]
@filipre: Thanks.
I fixed the multilabel links in devel and 2.7.

I will push a reworked version of the create-learner page soon.

**Comment by berndbischl** [comment]
Thx. Then close this here, pls
